scripts/setup-testbed.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -ex
```

Comment:
Looks like this script just sets up containers for the most part. It could be done with docker-compose in a more declarative way.

Reply:
yes this script is copied from brod where docker-compose was used.
but the setup is simple, so i'd rather keep it simple instead of adding another dependency.